compiler/ballerina-lang/src/main/java/io/ballerina/projects/BuildOptions.java

```diff
@@ -63,6 +63,7 @@ public boolean sticky() {
         return this.compilationOptions.sticky();
     }
 
+    @Deprecated
```

Comment:
Let's add a comment saying that it was deprecated because the language no longer has experimental features

Reply:
Added via https://github.com/ballerina-platform/ballerina-lang/pull/35835/commits/1d6a8f423d33c45b9a1955b2ada81c230c598926